src/extra/Markdown/index.tsx

```diff
@@ -138,11 +139,111 @@ type components = {
 	[element: string]: (props: any) => JSX.Element | React.ReactElement | Element;
 };
 
+interface Decorator {
+	match: RegExp;
+	component: string;
```

Comment:
Not sure how and where this is being used

Reply:
See the next commit.